eaf-pdf-viewer.el

```diff
@@ -167,55 +155,158 @@
     ("o" . "eaf-pdf-outline")
     ("T" . "toggle_trim_white_margin"))
   "The keybinding of EAF PDF Viewer."
-  :type 'cons)
+  :type '(alist :key-type (string :tag "Key bindings (e.g. \"C-n\", \"<f4>\", etc.)")
+                :value-type (string :tag "Python function"))
```

Comment:
Not just Python function, include JavaScript function and Elisp function.

Reply:
Ok, I fixed it to just `"Function name"` now